lib/Wikia/src/Service/Constants.php

```diff
@@ -9,4 +9,10 @@ class Constants {
 	const HELIOS_AUTH_HEADER = "X-Wikia-UserId";
 
 	const HTTP_STATUS_NO_CONTENT = 204;
+
+	const HTTP_STATUS_SERVER_ERROR = 500;
```

Comment:
Cool, I figured there must have been some existing class that had these, but it's pretty impossible to find. Thanks for pointing this out.